common/test/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ set(COMMON_TEST_SOURCE
         "${COMMON_TEST_SOURCE_DIR}/Renderer/AllocationTrackerTest.cpp"
         "${COMMON_TEST_SOURCE_DIR}/Renderer/CameraTest.cpp"
         "${COMMON_TEST_SOURCE_DIR}/Renderer/VertexTest.cpp"
+        "${COMMON_TEST_SOURCE_DIR}/View/AddNodesTest.cpp"
```

Comment:
This file is missing in the pr

Reply:
Oh it already exists, we just not building it I guess?

Reply:
Yes, exactly. I created it in the linked groups PR, but it wasn't added to `CMakeLists.txt`. Probably got lost on some rebase / fixup.